source/camera/CameraManager.h

```diff
@@ -48,6 +51,10 @@ enum class ViewModes : uint16_t
 
 class CameraManager
 {
+    friend class Console;
+    friend class CullingManager;
+    friend class ODFrameListener;
```

Comment:
IMHO, we should not use friend classes except for strongly linked classes. It is better to add setters/getters when you need. The problem with friend classes is that is makes hard to follow where a private variable is used (when you try to understand what some code does)